app/pipeline/rewriting_pipeline.py

```
@@ -0,0 +1,63 @@
+import logging
+from typing import Optional
+
+from langchain.output_parsers import PydanticOutputParser
+from langchain_core.prompts import (
+    ChatPromptTemplate,
+)
+
+from app.common.PipelineEnum import PipelineEnum
+from app.common.pyris_message import PyrisMessage, IrisMessageRole
+from app.domain.data.text_message_content_dto import TextMessageContentDTO
+from app.domain.data.competency_dto import Competency
```

Ruff (0.8.2)

`app.domain.data.competency_dto.Competency` imported but unused

Remove unused import: `app.domain.data.competency_dto.Competency`

(F401)